VogtPayroll5/PayrollConsoleReader.cs

```diff
@@ -8,21 +8,50 @@ class PayrollConsoleReader
     {
         public Employee GetEmployeeInfo()
         {
-            Employee emp = new Employee();
+            int empID;
+            string empName;
+            int hoursWorked;
```

Comment:
Technically, you don't need to declare these early. If you were looping through until the input was valid, then you would.

Reply:
Directly below you could do 

```c#
var empID = int.Parse(Console.ReadLine());
```